Docs/OpenAPISupport.md

```diff
@@ -0,0 +1,12 @@
+# OpenAPI Support
+
+The goal is to completely cover OpenAPI 3.x spec.
+
+Currently, the following features are **not** supported:
+
+- External References
+
+Some discrepancies with the OpenAPI spec are by design:
+
+- `allowReserved` keyword in parameters is ignored and all parameter values are percent-encoded
+- `allowEmptyValue` keyword in parameters is ignored as it's not recommended to be used
```

Comment:
This was in the README.md, I moved it here but didn't change it's contents (other than removing "upcoming")